### PR TITLE
Add cursor2D

### PR DESCRIPTION
**Pull request by chakravarthi589**
@ajpmaclean : Please Review.

### CODE REVIEW

**Comment by ajpmaclean** [comment]
@chakravarthi589 You will have conflicts for `src/Java.md` and possibly other files, so please resolve this for the outstanding merge requests. In a nutshell `src/Java/Miscellaneous` has been removed and these folders have been created `/src/Java/VisualizationAlgorithms`, `src/Java/Rendering` so some files have been renamed/removed. We are keeping the same layout as the C++ folders. From the logs you will see that we have been doing a major cleanup in preparation for a VTK merge request that should happen on Monday.

**Comment by chakravarthi589** [comment]
@ajpmaclean: I’ve resolved conflict.

**Comment by chakravarthi589** [comment]
@ajpmaclean: yeah am noticing all the changes made.

File: src/Java.md [hunk 1]
```diff
@@ -178,7 +178,11 @@ It would be appreciated if there are any Java VTK experts who could convert any
 [BackgroundColor](/Java/Visualization/BackgroundColor) |  |  Example program to set the background color of the render window
 [BackgroundGradient](/Java/Visualization/BackgroundGradient) |  | viewport having gradient background using the Background (bottom) and Background2 (top) colors.
 [ColorAnActor](/Java/Visualization/ColorAnActor) | vtkActor vtkCanvas | Switch style interaction adds C and A key actions for selecting Camera or Actor interaction, and T and J key actions for Trackball or Joystick interaction mode. Adds charEvent observer callback to restore the missing E and Q events to quit.
+
 [Cursor2D](/Java/Visualization/Cursor2D) | vtkCursor2D | Generates a 2D Cursor Representation.
+
+[Cutter](/Java/VisualizationAlgorithms/Cutter) | vtkCutter | How to use vtkCutter by cutting through a cube.
+
```
Comment (ajpmaclean) [inline src/Java.md]: There should be no blank lines here. Please remove three blank lines.

Comment (chakravarthi589) [reply src/Java.md]: Please check now

**Comment by chakravarthi589** [comment]
@ajpmaclean: please check now

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**